core/templates/dev/head/directives.js

```diff
@@ -205,3 +205,26 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
     }
   };
 }]);
+
+oppia.directive('scrollOnClick', function() {
+  return {
+    restrict: 'A',
+    scope: {
+      isDropdownOpen: '=isDropdownOpen',
+      isPrivate: '=isPrivate'
+    },
+    link: function(scope, element) {
+      var isDisabled;
+      scope.$watch('isDropdownOpen', function() {
+        isDisabled = scope.isDropdownOpen || scope.isPrivate;
+      });
+      element.on('click', function() {
+        if (!isDisabled) {
+          $('html, body').animate({
+            scrollTop: element.offset().top - 60
```

Comment:
Recommend you first assign the `60` to a well named variable, so readers understand its purpose better.